next_crm/patches/v1_0/update_won_date.py

```diff
@@ -0,0 +1,28 @@
+def execute():
+    from frappe import db, get_all
+    from frappe.utils import getdate, update_progress_bar
+
+    opportunity_list = get_all(
```

Comment:
If Opportunity has many values, then maybe we can run patch with a background job ?